App/localization/languages/es.json

```diff
@@ -0,0 +1,52 @@
+{
+  "error_screen_common_sorry": "Disculpa!\n",
+  "error_screen_error_cannot_load_cigarettes": "No logramos\ncargar tus\ncigarrillos.",
+  "error_screen_choose_other_location": "Elige otra ubicación",
+  "error_screen_error_description": "Existe un problema con nuestra base de datos, o no existen estaciones de monitoreo de aire cerca. Intente más tarde ",
+  "error_screen_error_message": "Error: {{errorText}}",
+  "home_station_too_far_message": "No logramos encontrar una estación cercana.\nLos resultados pueden ser inexactos a esta distancia.",
+  "home_share_title": "¿Sabía que puede estar fumando hasta 20 cigarrillos por día, solo por vivir en una gran ciudad?",
+  "home_share_message": "Maldición! 'hoy fumé'' {{cigarettes}} cigarrillos respirando el aire urbano. ¿Y tú? Encuentralo aquí: https://shootismoke.github.io",
+  "home_header_air_quality_station_distance": "Estación de calidad del aire: {{distanceToStation}}km de distancia",
+  "home_btn_why_is_station_so_far": "¿Por qué la estación está tan lejos??",
+  "home_btn_see_detailed_info": "Ver informacion detallada",
```

Comment:
On smaller screens, this text goes on 2 lines. Would it be possible to find a shorter variant?